src/app/game/game.component.html

```diff
@@ -396,17 +396,38 @@
               <td></td>
               <td></td>
             </tr>
+            <tr>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
+              <td></td>
```

Comment:
Why empty td ?

Reply:
in order to setup the style while waiting to do the real grid with typescript (we will generate the grid rather than put it directly in html)